frontend/src/metabase/auth/components/AuthLayout/AuthLayout.tsx

```diff
@@ -17,11 +16,17 @@ interface AuthLayoutProps {
 }
 
 export const AuthLayout = ({ children }: AuthLayoutProps): JSX.Element => {
-  const hasIllustration = useSelector(getHasIllustration);
+  const loginPageIllustration = useSelector(getLoginPageIllustration);
 
   return (
     <LayoutRoot>
-      {hasIllustration && <LayoutIllustration />}
+      {loginPageIllustration && (
+        <LayoutIllustration
+          data-testid="login-page-container"
+          backgroundImageSrc={loginPageIllustration}
+          isDefault={loginPageIllustration === "app/img/bridge.svg"}
```

Comment:
I don't feel good about this. I want 2 pieces of information here.
1. Whether or not the src is the default option because we render it based on the brand color
2. The source image of the custom option.

This works for now, but maybe I'll change the selector so it returns an object that has `isDefault` and `src` properties. Any thought about that?

Reply:
Fixed in 74349ee3110f4d4fd955348ab572f86685be87cb